src/dtos/monitor-span.dto.ts

```diff
@@ -250,7 +275,23 @@ export class MonitorSpanBaseDTO {
     example: propertyMetadata.monitorSpanDTOBeginDate.example,
     name: propertyMetadata.monitorSpanDTOBeginDate.fieldLabels.value,
   })
-  @IsNotEmpty()
+  @IsNotEmpty({
+    message: (args: ValidationArguments) => {
+      return CheckCatalogService.formatResultMessage('SPAN-8-A', {
+        fieldname: args.property,
+        key: KEY,
+      });
+    }
+  })
+  @IsInDateRange(MINIMUM_DATE, MAXIMUM_FUTURE_DATE, {
+    message: (args: ValidationArguments) => {
+      return CheckCatalogService.formatResultMessage('SPAN-8-B', {
+        fieldname: args.property,
+        date: args.value,
+        key: KEY,
+      });
+    }
+  })
   @IsIsoFormat({
     message: (args: ValidationArguments) => {
       return `${args.property} [ANALYZERRANGE-FATAL-A] The value : ${args.value} for ${args.property} must be a valid ISO date format yyyy-mm-dd`;
```

Comment:
use `CheckCatalogService.formatMessage` and remove the error code since those are not valid.